src/relation.js

```diff
@@ -374,6 +464,15 @@ export default RelationBase.extend({
   isInverse: function() {
     return (this.type === 'belongsTo' || this.type === 'morphTo');
   },
+  isForeignKeyTargeted() {
+    return this.foreignKeyTarget !== undefined;
```

Comment:
I would wollow the current code practice as on [line 452](https://github.com/tgriesser/bookshelf/pull/1397/files#diff-a9449d0672c5cc7f78c7fd45d3fd4da6R452):
```js
/*Not*/ return this.foreignKeyTarget !== undefined;
/*But*/ return (this.foreignKeyTarget != null); // which is weaker comparison and is true for non-null/non-undefined values
```